src/Event/ImageProcessingListener.php

```diff
@@ -77,6 +77,12 @@ public function implementedEvents() {
 	 * @return boolean
 	 */
 	protected function _autoRotate($imageFile, $format) {
+		// Auto-rotate applies only to JPEG files (i.e. not to PNG, GIF, etc..)
+		$format = strtolower($format);
+		if($format != 'jpg' && $format != 'jpeg') {
```

Comment:
Missing space, and weak comparison, should be !==.

Also can you explain the technical background why this is only working with jpeg?